charts/tempo/values.yaml

```diff
@@ -282,6 +286,7 @@ podAnnotations: {}
 podLabels: {}
 
 # Apply extra labels to common labels.
+# DEPRECATED: use global.commonLabels
```

Comment:
```suggestion

```
@ramon951 folks could still make use of extraLabels if there's a scenario where the labels might not needed to be applied to all components ? so I'd be more inclined towards keeping extraLabels in place yet enhancing the chart with the support of commonLabels which takes care of the scenario where the same labels are needed to be applied to all components.

Reply:
I was wrong. I'm fixing it